WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1810,7 +1810,11 @@ private void showPrepublishingNudgeBottomSheet() {
         if (fragment == null) {
             PrepublishingBottomSheetFragment prepublishingFragment =
                     PrepublishingBottomSheetFragment.newInstance(getSite(), mIsPage);
-            prepublishingFragment.show(getSupportFragmentManager(), PrepublishingBottomSheetFragment.TAG);
+
+            long delayMs = 100;
+            new Handler().postDelayed(
```

Comment:
Even though it won't happen very often (probably never), I believe this can lead to a leak of the EditPostActivity. We might want to consider creating the handler as a field and canceling it in `onDestroy`. Wdyt?

Reply:
Yes, that makes sense! That's a good practice 🙏

Reply:
Resolved in 54f61ef